.github/pull_request_template.md

```diff
@@ -1,21 +1,35 @@
-## Link to ticket:
```

Comment:
I think link to JIRA ticket or GH issue is a requirement we need to keep.

Reply:
I have used the autolink feature in GitHub so that each commit is linked to a Jira ticket. But good point.